modules/auxiliary/gather/enum_dns.rb

```diff
@@ -1,483 +1,432 @@
         This module can be used to gather information about a domain from a
         given DNS server by performing various DNS queries such as zone
         transfers, reverse lookups, SRV record bruteforcing, and other techniques.
-
-      },
-      'Author'		=> [ 'Carlos Perez <carlos_perez[at]darkoperator.com>' ],
-      'License'		=> MSF_LICENSE,
-      'References' 	=>
-        [
-          ['CVE', '1999-0532'],
-          ['OSVDB', '492'],
-        ]
-      ))
+    ),
+      'Author'         => [
+        'Carlos Perez <carlos_perez[at]darkoperator.com>',
+        'Nixawk'
+      ],
+      'License'        => MSF_LICENSE,
+      'References' 	   => [
+        ['CVE', '1999-0532'],
+        ['OSVDB', '492']
+      ]))
 
     register_options(
       [
-        OptString.new('DOMAIN', [ true, "The target domain name"]),
-        OptBool.new('ENUM_AXFR', [ true, 'Initiate a zone transfer against each NS record', true]),
-        OptBool.new('ENUM_TLD', [ true, 'Perform a TLD expansion by replacing the TLD with the IANA TLD list', false]),
-        OptBool.new('ENUM_STD', [ true, 'Enumerate standard record types (A,MX,NS,TXT and SOA)', true]),
-        OptBool.new('ENUM_BRT', [ true, 'Brute force subdomains and hostnames via the supplied wordlist', false]),
-        OptBool.new('ENUM_IP6', [ true, 'Brute force hosts with IPv6 AAAA records',false]),
+        OptString.new('DOMAIN', [true, 'The target domain']),
+        OptBool.new('ENUM_AXFR', [true, 'Initiate a zone transfer against each NS record', false]),
+        OptBool.new('ENUM_BRT', [true, 'Brute force subdomains and hostnames via the supplied wordlist', false]),
+        OptBool.new('ENUM_A', [true, 'Enumerate dns A record', true]),
+        OptBool.new('ENUM_CNAME', [true, 'Enumerate dns CNAME record', true]),
+        OptBool.new('ENUM_MX', [true, 'Enumerate dns MX record', true]),
+        OptBool.new('ENUM_NS', [true, 'Enumerate dns NS record', true]),
+        OptBool.new('ENUM_SOA', [true, 'Enumerate dns SOA record', false]),
+        OptBool.new('ENUM_TXT', [true, 'Enumerate dns TXT record', false]),
         OptBool.new('ENUM_RVL', [ true, 'Reverse lookup a range of IP addresses', false]),
-        OptBool.new('ENUM_SRV', [ true, 'Enumerate the most common SRV records', true]),
-        OptPath.new('WORDLIST', [ false, "Wordlist for domain name bruteforcing", ::File.join(Msf::Config.data_directory, "wordlists", "namelist.txt")]),
-        OptAddress.new('NS', [ false, "Specify the nameserver to use for queries (default is system DNS)" ]),
+        OptBool.new('ENUM_TLD', [true, 'Perform a TLD expansion by replacing the TLD with the IANA TLD list', false]),
+        OptBool.new('ENUM_SRV', [true, 'Enumerate the most common SRV records', false]),
+        OptBool.new('STOP_WLDCRD', [true, 'Stops bruteforce enumeration if wildcard resolution is detected', false]),
+        OptBool.new('STOP_STORE_LOOT', [true, 'Enable Default STORE_LOOT feature in metasploit', false]),
```

Comment:
I'd suggest renaming this option to `STORE_LOOT` and update the description to read like "Store acquired DNS records as loot"